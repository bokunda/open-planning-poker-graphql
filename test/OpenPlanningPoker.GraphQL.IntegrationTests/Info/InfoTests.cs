namespace OpenPlanningPoker.GraphQL.IntegrationTests.Info;

public class InfoTests
{
    [Fact]
    public async Task GetInfo_Success()
    {
        // Arrange
        const string query = "query { info() }";

        var builder = new ServiceCollection()
            .AddGraphQlWithSchema();

        // Act
        var result = await builder.ExecuteRequestAsync(query);

        // Assert
        result.ToJson().MatchSnapshot();
    }
}