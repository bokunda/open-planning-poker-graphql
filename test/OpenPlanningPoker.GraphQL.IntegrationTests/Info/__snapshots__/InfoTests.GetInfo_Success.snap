{
  "data": {
    "info": "v0.0.1"
  }
}
